Submitted By: BLFS Book <blfs-book@linuxfromscratch.org>
Date: 2003-10-04
Initial Package Version: 3.70
Origin: BLFS Book
Description: Fixes compilation.
diff -ur wvstreams-3.70.orig/src/crypto/wvcrypto.cc wvstreams-3.70/src/crypto/wvcrypto.cc
--- wvstreams-3.70.orig/src/crypto/wvcrypto.cc	2002-03-21 02:29:19.000000000 +0500
+++ wvstreams-3.70/src/crypto/wvcrypto.cc	2003-04-10 12:54:53.000000000 +0600
@@ -146,7 +146,8 @@
     int bufsize = ((hexbytes < 2048) ? 2048 : hexbytes) + 16;
     //int bufsize = hexbytes/2;
     
-    unsigned char *keybuf = new unsigned char[bufsize], *bufp;
+    unsigned char *keybuf = new unsigned char[bufsize];
+    const unsigned char *bufp;
     char *keystr;
     RSA *rp;
